dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+mpHealth=MicroProfile Health
+mpHealth.desc=Configuration for MicroProfile Health.
+
+fileUpdateInterval=Interval time for updating the file based health checks.
```

Comment:
```suggestion
fileUpdateInterval=Interval time at which to update the file-based health checks.
```

Reply:
what is getting "update" in tis context? Are you updating a file or the check? Or running a check and updating a file?

Reply:
It runs a check at intervals and updates a file if the checks are successful. The update consists of it _touching the file_ (i.e. modifies last accessed time). Hopefully the description entry explains it better. And it would appear its the description entry that is published on the reference documentation, and this value isn't used (but needed for